contrib-docs/how-to-write/what-we-write.md

```diff
@@ -0,0 +1,91 @@
+---
+title: What we write
+slug: <MDN/Contribute/how-to-write/what-we-write>
+tags:
+  - <Guide>
+  - <Guidelines>
+  - Writing
+  - Documentation
+  - MDN Meta
+---
+<{{MDNSidebar}}>
+
+In this article, you'll find information about whether or not a given topic and/or type of content should be included on MDN.
+
+> **Note:** Be aware that all contributions to MDN fall under specific open source licenses; these are [described in detail](/en-US/docs/MDN/About#copyrights_and_licenses) on our [About MDN](/en-US/docs/MDN/About) page. Keep in mind that Mozilla's [Websites & Communications Terms of Use](https://www.mozilla.org/en-US/about/legal/terms/mozilla/) are in effect when you use or contribute to MDN. <!--- links need to be revisited--->
+
+## Topics that belong on MDN
+
+In general, if it's an open web-facing technology, we document it on MDN. This includes any feature that can be used by web developers creating websites and applications, now and in the near future. If a feature is implemented by multiple browsers and either accepted as standard or is progressing towards standardization, then yes, we definitely document it here. If a feature is still very experimental and not implemented in multiple browsers and/or liable to change, then it is still suitable for inclusion, but may not be seen as a priority for the writer's team to work on.
+
+Our primary focus is to write about the following front-end web technologies:
+
+- [HTML](/en-US/docs/Web/HTML)
+- [CSS](/en-US/docs/Web/CSS)
+- [JavaScript](/en-US/docs/Web/JavaScript)
+- [SVG](/en-US/docs/Web/SVG)
+- [Web APIs](/en-US/docs/Web/API)
+- [WebGL](/en-US/docs/Web/API/WebGL_API)
+- etc.
+
+And so, you too can contribute towards these topic areas.
+
+> **Note:** Back-end technologies usually have their own documentation elsewhere that MDN does not attempt to supersede, although we [do have some exceptions](/en-US/docs/Learn/Server-side).
+
+We also welcome topics that cut across technologies but are relevant to web development, such as:
+
+- [Accessibility](/en-US/docs/Web/Accessibility)
+- [AJAX](/en-US/docs/Web/Guide/AJAX)
+- [Web graphics](/en-US/docs/Web/Guide/Graphics)
+- [Progressive web apps](/en-US/docs/Web/Progressive_web_apps)
+- [Web-based games](/en-US/docs/Games)
+
+> **Note:** MDN covers some non-standard features if they're exposed to the web, especially if they find common usage; for example, we have documentation for WebKit-specific CSS properties. MDN also covers some non-web standards technologies if they are considered useful enough to web developers (see our [Web-related technologies](/en-US/docs/Related) section).
```

Comment:
Honest I think we pull this entirely - we'll merge the list here with the one above